spack/local/packages/palace/package.py

```diff
@@ -161,9 +162,12 @@ class Palace(CMakePackage, CudaPackage, ROCmPackage):
     ## -- libxsmm --
     with when("+libxsmm"):
         # NOTE: @=main != @main since libxsmm has a version main-2023-22
-        depends_on("libxsmm@=main~shared blas=0")
+        depends_on("libxsmm@=main blas=0")
         depends_on("libxsmm+debug", when="build_type=Debug")
         depends_on("libceed+libxsmm")
+        # NOTE: libxsmm builds on MacOS have linker issues
+        # https://github.com/libxsmm/libxsmm/issues/883
+        depends_on("libxsmm~shared")
```

Comment:
@hughcars I think this should be `+shared` if I am reading https://github.com/awslabs/palace/blob/main/cmake/ExternalLIBXSMM.cmake#L26 correctly